### PR TITLE
Remove PropertyDDS packages which are not depended on by PropertyDDS

### PR DESCRIPTION
**Pull request by CraigMacomber**
## Description

Deletes some packages from experimental/PropertyDDS.

The DDS and its dependencies are kept, but the rest of its packages are not.

Delete the following published experimental packages:

- @fluid-experimental/property-shared-tree-interop
- @fluid-experimental/property-binder
- @fluid-experimental/property-proxy
- @fluid-experimental/property-inspector-table

Delete the following private packages:

- @fluid-example/property-inspector
- @fluid-example/schemas

Changes made by deleting the packages, then running:
```
pnpm i --no-frozen-lockfile
pnpm layer-check --md .
```

and manually adding a changeset.

## Breaking Changes

These packages will no longer be published:

- @fluid-experimental/property-shared-tree-interop
- @fluid-experimental/property-binder
- @fluid-experimental/property-proxy
- @fluid-experimental/property-inspector-table

## Reviewer Guidance

The review process is outlined on [this wiki page](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).

### CODE REVIEW

**Comment by tylerbutler** [comment]
Obligatory question: were you able to check with partners to see if this change would impact them?

File: .changeset/hand-picked-string.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+---
+---
+"section": "other"
+---
+Remove some PropertyDDS experimental packages.
```
Comment (tylerbutler) [inline .changeset/hand-picked-string.md:6]: nit

```suggestion
Remove some experimental PropertyDDS-related packages
```

File: .changeset/hand-picked-string.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+The following packages will no longer be published:
+
+- @fluid-experimental/property-shared-tree-interop
+- @fluid-experimental/property-binder
+- @fluid-experimental/property-proxy
+- @fluid-experimental/property-inspector-table
+
+PropertyDDS itself and its dependencies have been kept.
```
Comment (tylerbutler) [inline .changeset/hand-picked-string.md:15]:
```suggestion
PropertyDDS itself and its dependencies will continue to be published.
```

**Review by tylerbutler — APPROVED** [review]
Approving for docs team.

**Comment by CraigMacomber** [comment]
> Obligatory question: were you able to check with partners to see if this change would impact them?

Last Wednesday in our sync they were asked if they had any reason not to do this (and they didn't), and I'm giving them a week find any reasons they could not think of at the time. Should I receive no opposition by tomorrow by tomorrow, I think it makes sense to go ahead with this removal. None of these packages have been actively maintained, and I confirmed the two main customers have their own forks/alternatives to property inspector table which the independently maintain and would not be impacted by this one being removed. As for the other packages, no on expressed any opinions yet, which means no reason to keep them.

**Review by Abe27342 — APPROVED** [review]

**Comment by CraigMacomber** [comment]
Autodesk has requested a 2-3 day timeline extension to finish reviewing their use of these packages. moving the expected date for proceeding with this back to ~ Friday.

**Pull request merged**